src/Components/Server/src/ComponentHub.cs

```diff
@@ -152,7 +152,7 @@ public async Task<bool> ConnectCircuit(string circuitId)
 
                 circuitHost.InitializeCircuitAfterPrerender(CircuitHost_UnhandledException);
                 circuitHost.SetCircuitUser(Context.User);
-                circuitHost.SendPendingBatches();
+                circuitHost.ResumeCircuit();
```

Comment:
I guess I'm missing something, but what's `ResumeCircuit`? It's not in `master`, and doesn't appear to be defined in this PR. Where is it defined? Is it just a renaming of `SendPendingBatches`, or something else?

Reply:
This is leftover from a previous change.